### DISCUSSION TITLE
How would i go about updating my site to new version of the template?

### DISCUSSION THREAD

**Discussion by betarepos** [arthelokyo/astrowind#129] [Q&A] [answered]
Hello there...

Maybe a newbie question here. But i started using the Astrowind template last year and i love it. But how would i do an upgrade to the newer template version? What would be the best migration path for my content? Do i copy paste everything? Or is there a way to just npm upgrade?

Thanks for doing an awesome job. Love you all
Benny

**Comment by LamerManuel** [comment, marked as answer]
@betarepos 

I always do it like this (my Version atm is 0.9.8):

`npm update`

also works with pnpm or Yarn

**Reply by betarepos** [reply]
Thanks for the reply. 
Yes the npm update works for the packages, but what is with the changes in this repo? For example the announcement component is new, some menus changed, folder structure changed.
Is there a smart way to handle those? I read yesterday that there would be a way with forking the repository and then merging branches, but i haven´t worked with those mechanisms yet.

**Reply by LamerManuel** [reply]
like I said, it works for me. I updated today and it now shows me version 0.9.8 of astrowind.

**Reply by betarepos** [reply]
Thanks again, i guess i have to troubleshoot this then :)

**Reply by canomogollon** [reply]
i have  astro  4.2.6  and cant update  the template.... still in 1.0.0-beta.10
 
How to update the template on an existing site the command "npm update" updates the dependencies but not the template and the command "npx @astrojs/upgrade " updates the framework, any idea ? --- Thanks

**Reply by MrGKanev** [reply]
@betarepos I'm not sure he understood your question. I have the same problem and haven't found a solution to it.

**Comment by prototypa** [comment]
Hi @betarepos 

For now the only way is to do it manually, taking great care in each movement and we know that this is not optimal.

In the next *AstroWind v2* version we will take into account this important need to keep the template updated. To do this we will move much of the code we have now to independent dependencies, including all visual components and widgets. We are working on an initial draft of possible features.
